### PR TITLE
UG delete and flag notes wording bug

### LINKED ISSUE #193: [PE-D] Unclear wording "deleting an index"
Unclear what deleting an index means. It should be to delete the contact at that index.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: kerbysoh/ped#13

### PR DESCRIPTION
**Pull request by ryantanlien**
Resolves #193 and #192

https://ryantanlien.github.io/tp/UserGuide.html

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -291,10 +291,10 @@ Format:
 addskill INDEX [s/SKILL]...
 ```
 
-* Only valid indexes are edited. <br>
-  e.g If there are only `4` applicants in the app but `5` is specified, then the intended action will not be carried out.
-* Only positive indexes are edited. <br>
-  e.g As we label our applicants incrementally starting from `1` (e.g `1, 2, 3, and so on`), an index of `-1` will not be tagged to an applicant.
+* Index of applicant specified must not be more than the total number of applicants in LinkedOUT. <br>
+  e.g If there are only `4` applicants in the app but `5` is specified, then the addition will not be carried out as it is invalid.
+* Index of applicant specified must be a positive integer. <br>
```
Comment (RyanCheungJF) [inline docs/UserGuide.md:296]: Hmm I think we should probably change the term `integer` to `number` as integer is a more commonly used term in CS, but UG is meant for those without CS background.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -291,10 +291,10 @@ Format:
 addskill INDEX [s/SKILL]...
 ```
 
-* Only valid indexes are edited. <br>
-  e.g If there are only `4` applicants in the app but `5` is specified, then the intended action will not be carried out.
-* Only positive indexes are edited. <br>
-  e.g As we label our applicants incrementally starting from `1` (e.g `1, 2, 3, and so on`), an index of `-1` will not be tagged to an applicant.
+* Index of applicant specified must not be more than the total number of applicants in LinkedOUT. <br>
```
Comment (RyanCheungJF) [inline docs/UserGuide.md:294]: This does give more insight than the previous statement, but is there any way to simplify this? It seems like a very 'CS' way of thinking. `this must not violate cond1`. If I can think of a better alternative I will let you know.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -437,7 +437,7 @@ except after search and view commands. <br>
   and will no longer be displayed at the top.
   * e.g After a sort command, flagged applicants are not displayed at the top of the list and will be displayed according to the order 
   specified in the command. 
-  * e.g After a search command, applicants will be displayed in order of which applicants most satisfy the search conditions.
+  * e.g After a search command, applicants will be displayed in descending order of number of matches with the search conditions.
```
Comment (RyanCheungJF) [inline docs/UserGuide.md:440]: I think maybe an example could support this statement? If I were viewing this from the POV of an outsider, I think I might not fully understand this statement. Can check with @kaiyichen as I suggested her to add an example as well.

**Review by RyanCheungJF — APPROVED** [review]
LGTM, just a few minor suggestions to make the language more readable for a non-CS person.

File: docs/UserGuide.md (hunk 3 above)
Comment (ryantanlien) [reply docs/UserGuide.md]: Asked users to refer to search command for more information as example is already there.

File: docs/UserGuide.md (hunk 2 above)
Comment (ryantanlien) [reply docs/UserGuide.md:294]: Not sure if there's a more layman but clearer way to put this

File: docs/UserGuide.md (hunk 1 above)
Comment (ryantanlien) [reply docs/UserGuide.md]: Made the change

**Pull request merged**